addons/pos_hr/static/src/overrides/components/navbar/navbar.js

```diff
@@ -8,7 +8,10 @@ patch(Navbar.prototype, {
         }
 
         const { cashier } = this.pos;
-        return super.showCashMoveButton && (!cashier || cashier._role == "manager");
+        return (
+            super.showCashMoveButton &&
+            (!cashier || cashier._role == "manager" || cashier.user_id?.id === this.pos.user.id)
```

Comment:
Maybe you can use `employeeIsAdmin` so that we do not repeat parts of the code

Reply:
but then I guess we'll lose this part https://github.com/odoo/odoo/blob/8b6d2b1a77ed1a97d44a09058de86802e0db2df0/addons/pos_hr/static/src/overrides/components/navbar/navbar.js#L6-L8

which do some checks first
https://github.com/odoo/odoo/blob/8b6d2b1a77ed1a97d44a09058de86802e0db2df0/addons/point_of_sale/static/src/app/navbar/navbar.js#L54-L56

This was my thought process! Let me know what you think🙏🏻